babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -249,6 +265,8 @@ List<SqlCreateAttribute> CreateTableAttributes() :
             e = CreateTableAttributeFallback()
         |
             e = CreateTableAttributeJournalTable()
+        |
+            e = CreateTableAttributeChecksum()
         ) { list.add(e); }
     )+
```

Comment:
Would it be a valid query to have multiple checksums(don't know if it is)? If it is, it would be nice to also have test for multiple checksums.

Reply:
I believe there can only be one checksum attribute for a create table statement